### PR TITLE
[OSDOCS-10571]: What's New: Upgrading machine pools in ROSA

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.15
Issue:

https://issues.redhat.com/browse/OSDOCS-10571
Link to docs preview:

https://76093--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_release_notes/rosa-release-notes.html#rosa-q1-2024_rosa-whats-new
QE review:
- [ ] QE has approved this change.

This is a "What's New" addition based on https://issues.redhat.com/browse/OSDOCS-10179. **NO QE approval needed.** 
Additional information:

### CODE REVIEW

File: rosa_release_notes/rosa-release-notes.adoc [hunk 1]
```diff
@@ -66,6 +66,8 @@ For more information on region availabilities, see xref:../rosa_architecture/ros
 
 * **ROSA CLI update.** The ROSA CLI (`rosa`) was updated to a new version. For information about what has changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.35[ROSA CLI release notes]. For more information about the ROSA CLI (`rosa`), see xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc#rosa-about_rosa-getting-started-cli[About the ROSA CLI].
 
+* **Machinepool update.** You can now upgrade machinepools that are configured on ROSA with HCP clusters. For more information, see xref:../cli_reference/rosa_cli/rosa-manage-objects-cli.adoc#rosa-upgrade-machinepool_rosa-managing-objects-cli[upgrade machinepool].
```
Comment (aldiazRH) [inline rosa_release_notes/rosa-release-notes.adoc:69]:
```suggestion
* **Machine pool update.** You can now upgrade machine pools that are configured on ROSA with HCP clusters. For more information, see xref:../cli_reference/rosa_cli/rosa-manage-objects-cli.adoc#rosa-upgrade-machinepool_rosa-managing-objects-cli[upgrade machinepool].
```

**Review by aldiazRH — COMMENTED** [review]
Tiny change, but otherwise LGTM! No need to re-review after this change.

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**